### ISSUE TITLE
Difficulty Running Onsets and Frames Transcription

### ISSUE THREAD

**Issue by axiom-of-joy** [tensorflow/magenta#1542]
I am attempting to locally run onsets and frames transcription in Python3 on Ubuntu 18.04. I followed the instructions [here](https://github.com/tensorflow/magenta/tree/master/magenta/models/onsets_frames_transcription#transcription-script), manually setting up a Magenta environment on my laptop, downloading and unzipping the model checkpoint into a folder called `checkpoints`, and entering
```
onsets_frames_transcription_transcribe --model_dir="checkpoints/" temp.wav
```
with a test file `temp.wav` in the same folder. The terminal output and error messages can be seen [here](https://github.com/tensorflow/magenta/files/3243423/output.txt).

Thanks in advance for the help!

**Comment by cghawthorne** [comment]
I think we just need to use a six wrapper on one of those calls for py3 compatibility. I'll take a look.

**Comment by cghawthorne** [comment]
This should be fixed by af45e4ad2e1abadf74e3607d24a01fea523cddf1
